.editorconfig

```diff
@@ -0,0 +1,26 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+
+[*.json]
+indent_size = 2
```

Comment:
I've noticed that we use an `indent_size` in https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/.editorconfig#L16. Maybe we want to change this to be consistent with the starter template?